independent-publisher-connectors/One-Time Secret/apiProperties.json

```diff
@@ -0,0 +1,36 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "username": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Email",
+          "description": "The Email for this api",
+          "tooltip": "Provide the Email",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": true,
+            "required": "true"
+          }
+        }
+      },
+      "password": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key provided",
+          "tooltip": "Provide the API Key",
+          "constraints": {
+            "tabIndex": 3,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Aldo Gillone",
+    "stackOwner": "Aldo Gillone"
```

Comment:
@alduzzen1985 please change this to the API Owners name.
 "stackOwner": "Name of company or organization that connector is connecting to."